changelog/dmd.AArch64.dd

```diff
@@ -0,0 +1,28 @@
+# Add code generation for AArch64 CPU
+
+Adds the switch `-arm` which causes dmd to emit AArch64 code.
```

Comment:
As we agreed on in the meeting Friday, the switch should not be exposed to users yet.

Reply:
It remains undocumented as far as users are concerned.

Nobody can help with the development of it if they cannot turn it on.

Besides, it's a nuisance and impractical to keep altering the source code every time a push is done. Blech.

Reply:
The backend code can still get merged, but you agreed to Rikki's comment that the switch shouldn't be public remember? Advertising the `-arm` switch in the changelog is the opposite of that.

Reply:
ping @dkorpel

Reply:
Still unaddressed

Reply:
I'm going to leave the -arm switch in, and leave it undocumented, for reasons explained above.

Reply:
@thewilsonator asked for a changelog.

Reply:
documenting it would be putting it on the command line documentation, both what the compiler emits and the web page for it.

Reply:
We users read changelogs. It's important to keep up with what's changing in the tools you use. DMD's changelogs would be a lot more useful to us if they weren't treated like to-do lists (D has an issue tracker for that kind of thing, doesn't it?) littered with unfinished language features and compiler switches.

Reply:
Do you understand it's frustrating when in a DLF meeting we conclude "we're going to do X", then the next day you're doing Y instead, and refuse to address or even acknowledge this disrepancy?

Reply:
It remains undocumented in the compiler's list of switches and the documentation for how to use the compiler. It's entirely proper to have a changelog entry for a 6000 line addition.